tests/model_explainability/guardrails/conftest.py

```diff
@@ -133,3 +151,435 @@ def hap_detector_route(
         service=hap_detector_isvc.name,
         wait_for_resource=True,
     )
+
+
+@pytest.fixture(scope="class")
+def installed_tempo_operator(admin_client: DynamicClient, model_namespace: Namespace) -> Generator[None, Any, None]:
```

Comment:
All these fixtures could probably be used by other teams. Could you place them (in this or a follow-up PR) in /tests/fixtures/telemetry.py or similar? Thanks!